packages/payment-integration-api/src/payment/payment-method-actions.ts

```diff
@@ -0,0 +1,51 @@
+import { Action } from '@bigcommerce/data-store';
+
+import PaymentMethod from './payment-method';
```

Comment:
Is there a reason why we are adding these here?

Reply:
I added this file because some types are used in stripev3-payment-strategy.spec.ts file. 
Should I keep the whole file as it is now or leave just types which are used?

Reply:
I just had a look at the file, you can possibly rewrite the test to avoid this. I say since these are internals for core package and should not be exported or used outside.

Reply:
Tests are rewrited so the was no need to import this file